### ISSUE TITLE
Formatting does not work 

### ISSUE THREAD

**Issue by kanlukasz** [artagnon/vstwig#1]
It seems that formatting does not work

#### Example below (GIF Video):
![twig1](https://user-images.githubusercontent.com/30685349/119649330-c9477100-be22-11eb-846c-a67df77df1a1.gif)

#### Tested on:
```
Version: 1.56.2
Commit: 054a9295330880ed74ceaedda236253b4f39a335
Date: 2021-05-12T17:13:13.157Z
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Windows_NT x64 10.0.19042
```

#### Additional info:
After re-loading the VSCode, Dev Tools Console returns the following warning:
```
[twig]: language configuration: expected `comments.blockComment` to be an array of two strings.
languageConfigurationExtensionPoint.ts:152
```
![image](https://user-images.githubusercontent.com/30685349/119649671-2d6a3500-be23-11eb-9b5e-7d67f74eba99.png)

Let me know if you need more information or more tests

**Comment by artagnon** [comment]
Hi @kanlukasz,

Thank you for creating the issue. Can you give me the following information:

1. Does formatting work on any other twig document, or does VSTwig not work in general for you?
2. Can you try VSTwig with the following document?

```twig
{% extends 'front/layout.html.twig' %}

{% set title = ( 'Home'|trans) ~ ' — Apollon' %}

{% block body %}

    {% set params = {
        'defaultPost': {
            'users': [
                {
                    'name': 'Ama',
                    'image': asset('front/images/feed-default-ama.png'),
                    'image_retina': asset('front/images/feed-default-ama@2x.png')
                },
                {
                    'name': 'Hermann',
                    'image': asset('front/images/feed-default-hermann.png'),
                    'image_retina': asset('front/images/feed-default-hermann@2x.png')
                }
            ],
            'title': "Welcome Apollonians!"|trans,
            'subtitle': "Founder of Apollon"|trans,
            'date': "October 31, 2020 at 12h"|trans,
            'content': "Find here the people and stories that inspires you"|trans,
            'action': "Discover"|trans
        }
    } %}
    <feed-component :params="{{ params|json_encode }}" />

{% endblock %}
```

3. If (2) is successful, can you paste me the document that fails?
4. If (2) is unsuccessful, can you try adding the following to your settings.json, after making sure to remove any conflicting settings?

```json
  "[twig]": {
    "editor.defaultFormatter": "artagnon.vstwig"
  },
```

Thanks and warm regards.

**Comment by kanlukasz** [comment]
Hi @artagnon,
Thank you for your response. Below, I describe all the necessary details.
 ​​​​
> 1. Does formatting work on any other twig document, or does VSTwig not work in general for you?

The VSTwig formatter generally doesn't work on file that i tested yesterday. I'll check your sample file in a moment
 ​​​​

> 2. Can you try VSTwig with the following document?

![twig2](https://user-images.githubusercontent.com/30685349/119934686-7a1a5100-bf86-11eb-9f9b-4dff3cdf9201.gif)
Something happened (as you can see in the gif example), but now it's the wrong formatting
 ​​​​
> 3. Can you paste me the document that fails?

The file I tested that doesn't work is the original twig file from the Symfony Demo Blog from Github. It is here ---> https://github.com/symfony/demo/blob/main/templates/base.html.twig
 ​​​​
> 4. Can you try adding the following to your settings.json, after making sure to remove any conflicting settings?

I added this. Anyway conflict is impossible as I test your addon on a new and fresh install on VSCode without any addons or custom settings

**Comment by artagnon** [comment]
Hello @kanlukasz. Sorry for the delay, but I finally got around to looking at the issue this morning. It seems that the issue is Windows-specific, and has to do with hard-coding "\n" in several places. I've attempted to fix the issue now, although I don't have a Windows machine to test. Could you kindly report how vstwig 0.0.7 performs?

Warm regards.

**Comment by kanlukasz** [comment]
Hello

Unfortunately, it still doesn't work properly. Examples below

### Case 1 (tried with `CRLF` and `LF`):
![twig1](https://user-images.githubusercontent.com/30685349/142736499-bca4c357-d6d4-4292-89a1-02c4f6d1a130.gif)

### Case 2 (Formatting doesn't work at all when you have a twig comment at the top of the file):
![twig2](https://user-images.githubusercontent.com/30685349/142736525-1d1a5ded-abb8-417d-b88b-a3f755820111.gif)

### Case 3 (Even if it works, strange things still happen):
![twig3](https://user-images.githubusercontent.com/30685349/142736570-8198e19c-b8dc-4a43-a67d-c5240dd1bc58.gif)

### Case 4 (Strange things still happen - another example):
![twig4](https://user-images.githubusercontent.com/30685349/142736658-392a0fd0-da75-48e0-8260-e69ae1875b9d.gif)

### Case 5 (The formatter does not respect the indentation change [space | tabs]):
![twig5](https://user-images.githubusercontent.com/30685349/142736715-d6cb0689-fc53-456c-8ffc-6508621cdb74.gif)

**Comment by artagnon** [comment]
Hi @kanlukasz. Sorry for the long delay: I don't actually use Twig anymore, but found some time to fix the bugs nevertheless, as vstwig is a popular extension. Could you kindly verify that the issues are all fixed with 0.0.10?

**Comment by artagnon** [comment]
Closing due to lack of response. Please re-open if issue persists.
